src/lib/utils/strategies/FlagUnorderedStrategy.ts

```diff
@@ -8,13 +8,13 @@ export interface FlagStrategyOptions {
 	 * The accepted flags. Flags are key-only identifiers that can be placed anywhere in the command.
```

Comment:
Update TSDoc to mention what it means to set `string[]` and what it means to set `boolean`